<h1>Table of Contents<span class="tocSkip"></span></h1>
<div class="toc"><ul class="toc-item"><li><span><a href="#Задание" data-toc-modified-id="Задание-1"><span class="toc-item-num">1&nbsp;&nbsp;</span>Задание</a></span></li><li><span><a href="#Расшифровка-аудио" data-toc-modified-id="Расшифровка-аудио-2"><span class="toc-item-num">2&nbsp;&nbsp;</span>Расшифровка аудио</a></span></li><li><span><a href="#Запрос-для-ChatGPT" data-toc-modified-id="Запрос-для-ChatGPT-3"><span class="toc-item-num">3&nbsp;&nbsp;</span>Запрос для ChatGPT</a></span></li><li><span><a href="#Результаты" data-toc-modified-id="Результаты-4"><span class="toc-item-num">4&nbsp;&nbsp;</span>Результаты</a></span></li><li><span><a href="#Выводы" data-toc-modified-id="Выводы-5"><span class="toc-item-num">5&nbsp;&nbsp;</span>Выводы</a></span></li></ul></div>

## Задание

- Ознакомиться с звонками и самостоятельно оценить работу менеджера по трем выбранным критериям CQR.
- Написать такой промпт для ChatGPT, чтобы ИИ самостоятельно проанализировал диалог и сделал вывод.


## Расшифровка аудио

Для распознования текста будем использовать инструменты библиотеки SpeechRecognition и API Google Web Speech.

In [9]:
# импорт библиотек
import speech_recognition as sr 
import os 
from pydub import AudioSegment
from pydub.silence import split_on_silence
from nltk import word_tokenize

import warnings
warnings.filterwarnings("ignore")

In [2]:
# создадим объект для распознования
r = sr.Recognizer()

# зададим функцию для распознавания текста
def transcribe_audio(path):
    with sr.AudioFile(path) as source:
        audio_listened = r.record(source)
        text = r.recognize_google(audio_listened, language="ru-RU")
    return text

# функция для вывода большого текста
def large_transcribe_audio(path):
    sound = AudioSegment.from_file(path)  
    chunks = split_on_silence(sound,
        min_silence_len = 500,
        silence_thresh = sound.dBFS-14,
        keep_silence=500)
    folder_name = "audio-chunks"
    if not os.path.isdir(folder_name):
        os.mkdir(folder_name)
    whole_text = ""
    for i, audio_chunk in enumerate(chunks, start=1):
        chunk_filename = os.path.join(folder_name, f"chunk{i}.wav")
        audio_chunk.export(chunk_filename, format="wav")
        try:
            text = transcribe_audio(chunk_filename)
        except sr.UnknownValueError as e:
            print("неразборчиво")
        else:
            text = f"{text.capitalize()}. "
            print('реплика', ":", text)
            whole_text += text
    return whole_text

In [3]:
#получим первый текст
path = "audio3.wav"
text1 = large_transcribe_audio(path)

реплика : Оставайтесь на линии вам ответит первый свободный сотрудник. 
неразборчиво
неразборчиво
реплика : Александр суши. 
реплика : Добрый день. 
реплика : Как правильно попал я хотел узнать по отношению перетяжка руля. 
реплика : Как зовут александр. 
реплика : Во-первых это где-то находитесь осмотрелся в митино. 
реплика : Краснодар тургенева 26. 
неразборчиво
реплика : Краснодар улица гаврилова 26. 
реплика : На какой улице даже не знаю то что маленькая да ну это как бы центральная улица фестивальный микрорайон. 
реплика : Этанол. 
реплика : Барбитал. 
реплика : Ну как новый район кольцо-кольцо партизан аэродромная красных партизан. 
реплика : Огромное по времени. 
неразборчиво
реплика : Погода в хорошо примерно скажите надо так теперь не хочу сделать руль такой кожаный. 
реплика : Сколько это будет стоить чтобы знать. 
реплика : На него какая обычно него я типа chevrolet niva chevrolet niva. 
неразборчиво
неразборчиво
реплика : Сориентировать по стоимости материала. 
реплика : Г

In [4]:
# получим второй текст
path = "implantcity.wav"
text2 = large_transcribe_audio(path)

реплика : Скажите. 
реплика : Чтобы позвонить клиенту 0 чтобы прекратить вызов. 
реплика : Звонок лиза пожар. 
реплика : Начинаем звонить клиенту. 
неразборчиво
реплика : Страна. 
реплика : Елена. 
реплика : Стоматология простите меня светлана зовут подскажите оставили к нам. 
неразборчиво
неразборчиво
реплика : Да давайте на консультацию. 
реплика : Подскажите елена откуда к нам обращайтесь москва московская область. 
реплика : Московская область. 
реплика : Московская. 
реплика : Елена подскажите тоесть. 
реплика : Восстановлю. 
неразборчиво
реплика : Вообще-то у меня стояла на нижней челюсти мост 4 зуба на это и последняя сторона этот. 
реплика : Не держался этот мост вот и сейчас получается. 
реплика : Треснул вылетают полностью этот мост уже не держатся из-за того что корень зуба отвалился. 
реплика : Ну расскажи что-то салам я уже мучаюсь не знаю честно весь мост этот четыре зуба куда-то. 
реплика : Ну да ну вот тольятти чуть что-то надо делать а естественно есть у меня что учили

In [5]:
# получим третий текст
path = "whiteservice1.wav"
text3 = large_transcribe_audio(path)

реплика : Компания белая техника меня зовут александра. 
реплика : Здравствуйте подскажите iphone 14 + на 512 гб голубого цвета. 
реплика : Стас 512 голубой к сожалению сейчас такого наличии нет в челябинске можно могут привести с удалённого склада в течение 5 рабочих дней по предоплате 3.000 руб цена у него. 
реплика : 89990 руб за наличный расчёт оплаты по карте и переводом. 
реплика : Так ну тогда пару какой-то договор сделаешь предоплату вы можете мне столичном магазине соответственно получить чек внесения предоплаты. 
реплика : Какой товарный или кассовый какой. 
реплика : Статья точно вам не подскажу какой такой. 
неразборчиво
неразборчиво
реплика : Чекпоинт. 
реплика : Есть варианты всё вместе удалённо там будет о внесении только от вашего банка. 
реплика : Только мне именно такой привезут у меня какое-то будет подтверждение. 
реплика : Да мы создаём заявку для вас соответственно когда вы вносите предоплату вы её подтверждаете. 
реплика : Заявку отправил. 
реплика : Какой-то док

ChatGPT имеет ограничение по размеру промта, поэтому посчитаем количество токенов в итоговых текстах.

In [6]:
# посчитаем количество токенов 
len(word_tokenize(text1))

291

In [7]:
# посчитаем количество токенов 
len(word_tokenize(text2))

648

In [8]:
# посчитаем количество токенов 
len(word_tokenize(text3))

341

## Запрос для ChatGPT

Критерии, по которым можно оценить работу оператора:
1. вежливость;
2. речь;
3. инициатива;
4. проблема;
5. продукт;
6. совет;
7. возражения;
8. дожим;
9. контакты;
10. следующий шаг.

Возьмем три критерия, наиболее универсальные для трех звонков и ценим по шкале 0/0,5/1, где 0 - критерий отработан; 0,5 - была попытка, но не успешная; 0 - критерий не отработан.
1. вежливость - был ли оператор вежлив по время звонка?
2. проблема - выяснил ли оператор проблему клиента и подтвердил?
3. следующий шаг - было ли назначание следующего шага, клиента записали или подтвердили заказ?

**Таким образом, основной промт можно сформулировать следующим образом:** 
Оценить работу оператора по трем критериям: вежливость - был ли оператор вежлив по время звонка.
проблема - выяснил ли оператор проблему клиента. следующий шаг - клиента записали или подтвердили заказ. по шкале 0/0,5/1, где 0 - критерий отработан; 0,5 - была попытка, но не успешная; 0 - критерий не отработан. + текст диалога.

## Результаты

Для начала оценим самостоятельно работу менеджера.

Первый текст (перетяжка руля):
- Вежливость: 0,5 - оператор был вежлив, но перебивал клиента, недослушав его.
- Проблема: 1,0 - оператор выяснил проблему клиента с перетяжкой руля, рассказал о ценах.
- Следующий шаг: 0,5 - оператор спросил, когда клиенту было бы удобно приехать, но не назначил конкретный день.

Второй текст (стоматология):
- Вежливость: 1,0 - оператор была вежлива.
- Проблема: 0,5 - оператор выслушала проблему клиентки, но не задавала уточняющих вопросов, не подтвердила то, что расказала клиентка, а просто предложила консультацию.
- Следующий шаг: 1,0 - консультация назначена, критерий выполнен.

Третий текст (заказ телефона):
- Вежливость: 1,0 - оператор был вежлив.
- Проблема: 1,0 - оператор выслушал запрос клиентки, ответил на все вопросы (хоть и растерялся при ответе про чеки), но предложил разные варианты.
- Следующий шаг: 0 - заказ не был принят.

Для формирования запроса удалим из текста паузы и скорректируем реплики, которые модель не смогла распознать.

**Первый текст**

Оценить работу оператора по трем критериям: вежливость - был ли оператор вежлив по время звонка. проблема - выяснил ли оператор проблему клиента. следующий шаг - клиента записали или подтвердили заказ. по шкале 0/0,5/1, где 0 - критерий отработан; 0,5 - была попытка, но не успешная; 0 - критерий не отработан.
<br>реплика : Александр слушаю. 
<br>реплика : Добрый день. 
<br>реплика : Как правильно попал я хотел узнать по отношению перетяжка руля. 
<br>реплика : Как зовут александр. 
<br>реплика : Во-первых это где-то находитесь?
<br>реплика : Краснодар улица гаврилова 26. 
<br>реплика : На какой улице даже не знаю то что маленькая да ну это как бы центральная улица фестивальный микрорайон. 
<br>реплика : Ну как новый район кольцо партизан аэродромная красных партизан. 
<br>реплика : хорошо примерно скажите надо так теперь не хочу сделать руль такой кожаный. Сколько это будет стоить чтобы знать? 
<br>реплика : На него какая обычно него я типа chevrolet niva. 
<br>реплика : Сориентировать по стоимости материала. 
<br>реплика : Есть экокожа, кожа. 
<br>реплика : Ну я не знаю мне для продажи. 
<br>реплика : Ну нет а чего есть 3.000-4.000 я говорю на chevrolet ниве. 
<br>реплика : Я просто разница по стоимости будет. 
<br>реплика : Для мастера разница есть сложность работ. 
<br>реплика : Перетяжка руля экокожей. 
<br>реплика : Экокожа фактурная будет стоить 3.500. Будет 24000 руб. если будет кожа натуральная фактура. 
<br>реплика : Будет строить 5561 зависимости от материала. 
<br>реплика : Если таблетка натуральная гладкая либо 7 либо 7.500. 
<br>реплика : Короче завтра я подъеду да ещё расскажи адрес. 
<br>реплика : Улица гаврилова 20. 
<br>реплика : Гаврилова 26. 
<br>реплика : С 9:00 утра и до 19:00 в торговый зал написала 17:00 вечера до понедельника по субботу включительно. 
<br>реплика : Воскресенье выходной. 
<br>реплика : Спасибо спасибо когда планируете к нам. 
<br>реплика : На неделе наверное сейчас я сегодня не могу. 
<br>реплика : Хорошо подъезжайте поможем подскажем материалы. 
<br>реплика : Спасибо спасибо подъеду. 


**ответ ChatGPT**
- Вежливость: 1 (критерий отработан) - оператор был вежлив и проявлял участие во время звонка.
- Проблема: 1 (критерий отработан) - оператор выяснил проблему клиента с перетяжкой руля и предоставил информацию о стоимости различных материалов.
- Следующий шаг: 0,5 (была попытка, но не успешная) - оператор указал адрес офиса и расписание работы, но не уточнил, когда клиент планирует прийти.

**Второй текст**

Оценить работу оператора по трем критериям: вежливость - был ли оператор вежлив по время звонка. проблема - выяснил ли оператор проблему клиента. следующий шаг - было ли назначание следующего шага, клиента записали или подтвердили заказ. по шкале 0/0,5/1, где 0 - критерий отработан; 0,5 - была попытка, но не успешная; 0 - критерий не отработан.
<br>реплика : Стоматология простите меня светлана зовут подскажите оставили к нам.
<br>реплика : Да давайте на консультацию.
<br>реплика : Подскажите елена откуда к нам обращайтесь москва московская область.
<br>реплика : Московская область.
<br>реплика : Елена подскажите тоесть.
<br>реплика : Вообще-то у меня стояла на нижней челюсти мост 4 зуба на это и последняя сторона этот. Не держался этот мост вот и сейчас получается. Треснул вылетают полностью этот мост уже не держатся из-за того что корень зуба отвалился. Я уже мучаюсь не знаю честно весь мост этот четыре зуба куда-то. Ну да ну вот чуть что-то надо делать а естественно есть у меня что учились то нижняя строилась на этом мосту была его не будешь надо что-то думать что делать.
<br>реплика : Елена смотрите у нас просто клиника находится в москве есть две у нас на этом артистка и украинский бульвар 5 марксистская прямо метро. Недалеко 5 минут ходьбы. Украинский бульвар дом 6. В любом случае конечно чтобы мы вам помогли в вашей ситуации. Смотрите у нас бесплатно составление плана лечения. Хирург-имплантолог ортопед. Врачи 15 и 10 лет стажа. Вот скажите пожалуйста мы работаем без выходных с 8:00 до. Когда будет удобно вам приехать.
<br>реплика : Ну как можно на последний наверное время.
<br>реплика : Сегодня сможете?
<br>реплика : Сегодня я не смогу потому что у нас вот-вот машину отремонтируют мужу на ремонте стоит я не знаю когда обещали вот день-два ну можно там вообще ничего не знаю сидим ждём машину прогулка и выборгском конечно мне лучше на машине.
<br>реплика : Скажите пожалуйста на какой день недели.
<br>реплика : Через недельку. Где-то так.
<br>реплика : Ну скажите просто вам будни или выходной просто чтобы вам было комфортно там приехать.
<br>реплика : В принципе мне всё равно я на тебя работаю каждый день на работе у меня там. Поэтому нежелательно или в вечернее время потому что я я бы вас не работаю мне желательно в я могу закрыться уйти конечно в такое время ну а мне всё равно что будешь выходной у меня мучить не одинаковые.
<br>реплика : Я поняла.
<br>реплика : Смотрите конечно могу вам предложить это наверное. Киевская кольцевая. Подъехать. Строгино. Что ближе.
<br>реплика : Я просто в москву недавно московской области.
<br>реплика : Vegas где вот этот как его сейчас скажу.
<br>реплика : Получается у нас ближайшая спутник.
<br>реплика : На автобусе можно доехать до метро беляево.
<br>реплика : Крокус-сити вот этот вот. Вегас.
<br>реплика : Это если. Мякинино.
<br>реплика : Никитина никитин я путаю.
<br>реплика : Всё-равно нам ориентироваться с вами.
<br>реплика : Да вот смотрите если мякинина киевская то есть этот. Минут от этого метро. Ехать до усинска. Марксистская.
<br>реплика : Замечательно.
<br>реплика : Смотрите то есть могу если предложить вам. Понедельник как вам скажи 19.
<br>реплика : Давайте 19.
<br>реплика : Смотрите могу 17. Предложить.
<br>реплика : Свободна а вот это самое последнее.
<br>реплика : Ну у нас идёт консультация. Самое позднее 20.
<br>реплика : Вот давайте 20.
<br>реплика : Нормально булатова удобно.
<br>реплика : Да конечно сразу скажите мне пожалуйста вашу фамилию имя отчество.
<br>реплика : Винничук как винни-пух только чук Елена анатольевна.
<br>реплика : Звоним напоминаем.
<br>реплика : Вообще замечательно.
<br>реплика : Чтобы. Познакомлились с нашими врачами. Это немаловажно. Бюджетный вариант и средней стоимости если тоже.
<br>реплика : Хорошо я пойду. Спасибо большое.
<br>реплика : До свидания.


**ответ ChatGPT**
- Вежливость: 1 - оператор был вежлив по время звонка.
- Проблема: 1 - оператор выяснил проблему клиента.
- Следующий шаг: 1 - оператор назначил следующий шаг, записал клиента и подтвердил заказ.

**Третий текст**

Оценить работу оператора по трем критериям: вежливость - был ли оператор вежлив по время звонка. проблема - выяснил ли оператор проблему клиента. следующий шаг - клиента записали или подтвердили заказ. по шкале 0/0,5/1, где 0 - критерий отработан; 0,5 - была попытка, но не успешная; 0 - критерий не отработан.
<br>реплика : Компания белая техника меня зовут александра. 
<br>реплика : Здравствуйте подскажите iphone 14 + на 512 гб голубого цвета. 
<br>реплика : Стас 512 голубой к сожалению сейчас такого наличии нет в челябинске можно могут привести с удалённого склада в течение 5 рабочих дней по предоплате 3.000 руб цена у него. 
<br>реплика : 89990 руб за наличный расчёт оплаты по карте и переводом. 
<br>реплика : Так ну тогда пару какой-то договор сделаешь предоплату вы можете мне столичном магазине соответственно получить чек внесения предоплаты. 
<br>реплика : Какой товарный или кассовый какой. 
<br>реплика : Статья точно вам не подскажу какой такой. 
<br>реплика : Есть варианты всё вместе удалённо там будет о внесении только от вашего банка. 
<br>реплика : Только мне именно такой привезут у меня какое-то будет подтверждение. 
<br>реплика : Да мы создаём заявку для вас соответственно когда вы вносите предоплату вы её подтверждаете. 
<br>реплика : Заявку отправил. 
<br>реплика : Какой-то документ даёте или она где-то отображается. 
<br>реплика : Это отображается только в базе у нас. 
<br>реплика : Да откуда вы проведёте в течение 5 дней склад в москве находится. 
<br>реплика : Ну он он приедет в течение 5 дней или как там ну настройка обозначенная до священника 5 рабочих дней с момента оплаты. 
<br>реплика : 3.000. 
<br>реплика : 3000руб давно ещё подскажите вот 89990 цена не поменяется за эти как бы за это время то есть она не изменится никакой. 
<br>реплика : В данном случае цена фиксированная. 
<br>реплика : Но если произойдёт так что товар который будет на складе он будет именно новый поступать на склад для. 
<br>реплика : И цена вот момент изменится-то volkswagen свяжется предварительное. 
<br>реплика : Слива цена ниже может стать либо выше нас. 
<br>реплика : Такое случится. 
<br>реплика : Про вас предупредить соответствовать будет возможно. 
<br>реплика : В случае отказа вот эти вот 3001 предоплата возвращается всё поняла то как-то работаете когда к вам можно приехать с 10:00 утра до 8:00 вечера каждый день энгельса 30 энгельса 75 и университета можно подъехать. 
<br>реплика : А привезут какой его. 
<br>реплика : Станция другой. 
<br>реплика : Всё поняла спасибо вы лучше заранее перед тем как поехать позвоните мы оформим заявку для вас и подготовить заявку можно. 
<br>реплика : Спасибо всего доброго. 

**ответ ChatGPT**
- Вежливость: 1 (оператор был вежлив по время звонка)
- Проблема: 1 (оператор выяснил проблему клиента)
- Следующий шаг: 0 (заказ не был подтвержден или записан)

## Выводы

По результатам формирования запросов можно составить итоговую таблицу результатов

| Промт | Оценка человеческая | Оценка ИИ |
| :- | :-: | :-: |
| Оценить работу оператора по трем критериям: вежливость - был ли оператор вежлив по время звонка. проблема - выяснил ли оператор проблему клиента. следующий шаг - клиента записали или подтвердили заказ. по шкале 0/0,5/1, где 0 - критерий отработан; 0,5 - была попытка, но не успешная; 0 - критерий не отработан. реплика : Александр слушаю. реплика : Добрый день. реплика : Как правильно попал я хотел узнать по отношению перетяжка руля. реплика : Как зовут александр. реплика : Во-первых это где-то находитесь? реплика : Краснодар улица гаврилова 26. реплика : На какой улице даже не знаю то что маленькая да ну это как бы центральная улица фестивальный микрорайон. реплика : Ну как новый район кольцо партизан аэродромная красных партизан. реплика : хорошо примерно скажите надо так теперь не хочу сделать руль такой кожаный. Сколько это будет стоить чтобы знать? реплика : На него какая обычно него я типа chevrolet niva. реплика : Сориентировать по стоимости материала. реплика : Есть экокожа, кожа. реплика : Ну я не знаю мне для продажи. реплика : Ну нет а чего есть 3.000-4.000 я говорю на chevrolet ниве. реплика : Я просто разница по стоимости будет. реплика : Для мастера разница есть сложность работ. реплика : Перетяжка руля экокожей. реплика : Экокожа фактурная будет стоить 3.500. Будет 24000 руб. если будет кожа натуральная фактура. реплика : Будет строить 5561 зависимости от материала. реплика : Если таблетка натуральная гладкая либо 7 либо 7.500. реплика : Короче завтра я подъеду да ещё расскажи адрес. реплика : Улица гаврилова 20. реплика : Гаврилова 26. реплика : С 9:00 утра и до 19:00 в торговый зал написала 17:00 вечера до понедельника по субботу включительно. реплика : Воскресенье выходной. реплика : Спасибо спасибо когда планируете к нам. реплика : На неделе наверное сейчас я сегодня не могу. реплика : Хорошо подъезжайте поможем подскажем материалы. реплика : Спасибо спасибо подъеду.| 2.0 | 2.5 |
| Оценить работу оператора по трем критериям: вежливость - был ли оператор вежлив по время звонка. проблема - выяснил ли оператор проблему клиента. следующий шаг - было ли назначание следующего шага, клиента записали или подтвердили заказ. по шкале 0/0,5/1, где 0 - критерий отработан; 0,5 - была попытка, но не успешная; 0 - критерий не отработан. реплика : Стоматология простите меня светлана зовут подскажите оставили к нам. реплика : Да давайте на консультацию. реплика : Подскажите елена откуда к нам обращайтесь москва московская область. реплика : Московская область. реплика : Елена подскажите тоесть. реплика : Вообще-то у меня стояла на нижней челюсти мост 4 зуба на это и последняя сторона этот. Не держался этот мост вот и сейчас получается. Треснул вылетают полностью этот мост уже не держатся из-за того что корень зуба отвалился. Я уже мучаюсь не знаю честно весь мост этот четыре зуба куда-то. Ну да ну вот чуть что-то надо делать а естественно есть у меня что учились то нижняя строилась на этом мосту была его не будешь надо что-то думать что делать. реплика : Елена смотрите у нас просто клиника находится в москве есть две у нас на этом артистка и украинский бульвар 5 марксистская прямо метро. Недалеко 5 минут ходьбы. Украинский бульвар дом 6. В любом случае конечно чтобы мы вам помогли в вашей ситуации. Смотрите у нас бесплатно составление плана лечения. Хирург-имплантолог ортопед. Врачи 15 и 10 лет стажа. Вот скажите пожалуйста мы работаем без выходных с 8:00 до. Когда будет удобно вам приехать. реплика : Ну как можно на последний наверное время. реплика : Сегодня сможете? реплика : Сегодня я не смогу потому что у нас вот-вот машину отремонтируют мужу на ремонте стоит я не знаю когда обещали вот день-два ну можно там вообще ничего не знаю сидим ждём машину прогулка и выборгском конечно мне лучше на машине. реплика : Скажите пожалуйста на какой день недели. реплика : Через недельку. Где-то так. реплика : Ну скажите просто вам будни или выходной просто чтобы вам было комфортно там приехать. реплика : В принципе мне всё равно я на тебя работаю каждый день на работе у меня там. Поэтому нежелательно или в вечернее время потому что я я бы вас не работаю мне желательно в я могу закрыться уйти конечно в такое время ну а мне всё равно что будешь выходной у меня мучить не одинаковые. реплика : Я поняла. реплика : Смотрите конечно могу вам предложить это наверное. Киевская кольцевая. Подъехать. Строгино. Что ближе. реплика : Я просто в москву недавно московской области. реплика : Vegas где вот этот как его сейчас скажу. реплика : Получается у нас ближайшая спутник. реплика : На автобусе можно доехать до метро беляево. реплика : Крокус-сити вот этот вот. Вегас. реплика : Это если. Мякинино. реплика : Никитина никитин я путаю. реплика : Всё-равно нам ориентироваться с вами. реплика : Да вот смотрите если мякинина киевская то есть этот. Минут от этого метро. Ехать до усинска. Марксистская. реплика : Замечательно. реплика : Смотрите то есть могу если предложить вам. Понедельник как вам скажи 19. реплика : Давайте 19. реплика : Смотрите могу 17. Предложить. реплика : Свободна а вот это самое последнее. реплика : Ну у нас идёт консультация. Самое позднее 20. реплика : Вот давайте 20. реплика : Нормально булатова удобно. реплика : Да конечно сразу скажите мне пожалуйста вашу фамилию имя отчество. реплика : Винничук как винни-пух только чук Елена анатольевна. реплика : Звоним напоминаем. реплика : Вообще замечательно. реплика : Чтобы. Познакомлились с нашими врачами. Это немаловажно. Бюджетный вариант и средней стоимости если тоже. реплика : Хорошо я пойду. Спасибо большое. реплика : До свидания.| 2.5 | 3.0 |
| Оценить работу оператора по трем критериям: вежливость - был ли оператор вежлив по время звонка. проблема - выяснил ли оператор проблему клиента. следующий шаг - клиента записали или подтвердили заказ. по шкале 0/0,5/1, где 0 - критерий отработан; 0,5 - была попытка, но не успешная; 0 - критерий не отработан. реплика : Компания белая техника меня зовут александра. реплика : Здравствуйте подскажите iphone 14 + на 512 гб голубого цвета. реплика : Стас 512 голубой к сожалению сейчас такого наличии нет в челябинске можно могут привести с удалённого склада в течение 5 рабочих дней по предоплате 3.000 руб цена у него. реплика : 89990 руб за наличный расчёт оплаты по карте и переводом. реплика : Так ну тогда пару какой-то договор сделаешь предоплату вы можете мне столичном магазине соответственно получить чек внесения предоплаты. реплика : Какой товарный или кассовый какой. реплика : Статья точно вам не подскажу какой такой. реплика : Есть варианты всё вместе удалённо там будет о внесении только от вашего банка. реплика : Только мне именно такой привезут у меня какое-то будет подтверждение. реплика : Да мы создаём заявку для вас соответственно когда вы вносите предоплату вы её подтверждаете. реплика : Заявку отправил. реплика : Какой-то документ даёте или она где-то отображается. реплика : Это отображается только в базе у нас. реплика : Да откуда вы проведёте в течение 5 дней склад в москве находится. реплика : Ну он он приедет в течение 5 дней или как там ну настройка обозначенная до священника 5 рабочих дней с момента оплаты. реплика : 3.000. реплика : 3000руб давно ещё подскажите вот 89990 цена не поменяется за эти как бы за это время то есть она не изменится никакой. реплика : В данном случае цена фиксированная. реплика : Но если произойдёт так что товар который будет на складе он будет именно новый поступать на склад для. реплика : И цена вот момент изменится-то volkswagen свяжется предварительное. реплика : Слива цена ниже может стать либо выше нас. реплика : Такое случится. реплика : Про вас предупредить соответствовать будет возможно. реплика : В случае отказа вот эти вот 3001 предоплата возвращается всё поняла то как-то работаете когда к вам можно приехать с 10:00 утра до 8:00 вечера каждый день энгельса 30 энгельса 75 и университета можно подъехать. реплика : А привезут какой его. реплика : Станция другой. реплика : Всё поняла спасибо вы лучше заранее перед тем как поехать позвоните мы оформим заявку для вас и подготовить заявку можно. реплика : Спасибо всего доброго.| 2.0 | 2.0 |